src/js/App/Header/AppFilter/useAppFilter.js

```diff
@@ -127,7 +126,7 @@ const useAppFilter = () => {
         ...prev,
         isLoading: true,
       }));
-      let bundles = requiredBundles.filter((app) => !existingSchemas.includes(app)).map((app) => navigationFileMapper[app]);
+      let bundles = requiredBundles.filter((app) => !existingSchemas.includes(app));
```

Comment:
This still requires the `-navigation.json` partial, otherwise we'll fetch `/config/chrome/insights` which would be 404.

Reply:
Good catch, fixed!